system/ui/widgets/option_dialog.py

```diff
@@ -1,83 +1,70 @@
 import pyray as rl
-
-from openpilot.system.ui.lib.application import Widget
+from openpilot.system.ui.lib.application import Widget, FontWeight
 from openpilot.system.ui.lib.button import gui_button, ButtonStyle, TextAlignment
 from openpilot.system.ui.lib.label import gui_label
 from openpilot.system.ui.lib.scroll_panel import GuiScrollPanel
 
+# Constants
+MARGIN = 50
+TITLE_FONT_SIZE = 70
+ITEM_HEIGHT = 135
+BUTTON_SPACING = 50
+BUTTON_HEIGHT = 160
+ITEM_SPACING = 50
+LIST_ITEM_SPACING = 25
```

Comment:
fyi we don't always need to pull these out. only do it if it makes sense (like reuse)